### PR TITLE
Chapter1/Homework/1And2

### PR DESCRIPTION
**Pull request by OrangeCluster**
Finished the first homework and hopefully I'm doing this correctly. Hopefully also not way too late to start these home work assignments.

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,24 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Declare variables right next to where you use them.

Comment (Almantask) [reply Src/BootCamp.Chapter/Program.cs]: This means not declaring all the variables in one place.

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,24 @@ class Program
...
+            int age;
+            double weight;
+            double height;
+            double BMI;
+            Console.WriteLine("BMI Calculator");
+            Console.WriteLine("Please enter your name: ");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:11]: right now, code looks like a blob. Try to be more subtle and improve readability by separating unrelated code parts with a blank line.

For example, unrelated parts are:
Read age
Read name
Read weight
Calculate bmi

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (OrangeCluster) [reply Src/BootCamp.Chapter/Program.cs]: Thank you for looking this over. Bad habits left over from coding 15 years ago... I've adjusted it following your recommendations.

File: Src/BootCamp.Chapter/Program.cs (hunk 2 above)
Comment (OrangeCluster) [reply Src/BootCamp.Chapter/Program.cs:11]: Done. Thank you for the suggestions.

**Review by Almantask — APPROVED** [review]
Looks great now
